engine/src/main/java/com/sandboni/core/engine/sta/connector/HttpTemplateConnector.java

```diff
@@ -67,4 +95,43 @@ public boolean isMatch(String callerTemplate, String calleeTemplate) {
 
         return false;
     }
+
+    private Vertex createVertex(JsonEntry entry) {
+        if (CUCUMBER_ENTRY.equals(entry.getType())) {
+            return new CucumberVertex.Builder(entry.getClassName(), entry.getTestName())
+                    .withFeaturePath(entry.getFilepath())
+                    .withScenarioLine(Integer.parseInt(entry.getLineNumber()))
+                    .markAsExternalLocation()
+                    .build();
+        }else {
+            return new TestVertex.Builder(entry.getClassName(), entry.getTestName())
+                    .withFilePath(entry.getFilepath())
+                    .markAsExternalLocation()
+                    .build();
+        }
+    }
+
+    private JsonEntry[] getEntriesFromSeloniFile() {
+        String filePath = System.getProperty(SystemProperties.SELONI_FILEPATH.getName());
+        if (!Objects.isNull(filePath)){
+            log.info("'seloni.filepath' set to '{}'", filePath);
+            File file = new File(filePath);
+            if (file.exists()){
+                return parseFile(file);
+            }
+            log.warn("No Seloni file was found in: {} ", filePath);
```

Comment:
If the file set is not found, then should be better to throw and Exception as this would be an invalid input

Reply:
agree. fixed